include/exadg/structure/spatial_discretization/operators/linear_operator.cpp

```diff
@@ -35,11 +35,14 @@ LinearOperator<dim, Number>::do_cell_integral(IntegratorCell & integrator) const
 
   for(unsigned int q = 0; q < integrator.n_q_points; ++q)
   {
-    // engineering strains (material tensor is symmetric)
-    tensor const gradient = integrator.get_gradient(q);
+    // engineering strains
+    tensor const symmetric_gradient = integrator.get_symmetric_gradient(q);
```

Comment:
Should I revert to just `gradient`? This works due to the specific implementation, e.g., 
`S[0][1] = f2 * (strain_measure[0][1] + strain_measure[1][0]);`
for `dim = 2`, right?
Should I revert this adaptation for performance reasons @kronbichler ?